### PR TITLE
basic image upload and view with active storage

### PR DESCRIPTION
**Pull request by sabrina-ulbrich**

### CODE REVIEW

**Comment by sabrina-ulbrich** [comment]
- [ ] remove image
- [ ] adjust image size
- [ ] add aws
- [ ] migrate carrierwave pictures
- [ ] delete carrierwave stuff
- [ ] add tests

**Comment by sabrina-ulbrich** [comment]
Resizing of images is not perfect. There should be a better solution in Rails 6.0. So maybe it's better to upgrade rails than to put much effort in fixing this version.

File: app/controllers/image_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ImageController < ApplicationController
+  
+  def destroy
+    @image = ActiveStorage::Attachment.find(params[:id])
```
Comment (tyranja) [inline app/controllers/image_controller.rb]: Remove the @

**Comment by sabrina-ulbrich** [comment]
@tyranja I can now display the image format (square) without Rails 6.0. I don't know if we want to use it now, but it was definitely interesting to find out how it works (just a short method in the profile model). Now all new images are processed with ActiveStorage and the existing ones with carrierwave. Maybe we can play around on staging?

**Comment by tyranja** [comment]
Let's try that next week :+1: Great job!

**Comment by tyranja** [comment]
I have finally deployed it to staging and it seems to work:
aws_staging

So we can maybe merge and deploy it to production today? and then make a new issue for exporting the other pics? @sabrina-ulbrich Maybe we can combine that with adding aws to the mwg, than we work in the same scope.

**Comment by sabrina-ulbrich** [comment]
Yeah, let's talk about that today evenig. I collected some links related to the export pictures issue, but there are different approaches and I didn't do this before so it's better to look at this together @tyranja

**Comment by sabrina-ulbrich** [comment]
Here are some explanations for migrating pics from carrierwave to active storage:
https://www.stefanwienert.de/blog/2018/11/05/activestorage-migrating-from-carrierwave-attachment-pointers/
step 3 in the first answer: https://stackoverflow.com/questions/52528623/migrating-carrierwave-to-activestorage
Description for paperclip (unfortunately carrierwave doesn't have it), but it explains the general approach: https://github.com/thoughtbot/paperclip/blob/master/MIGRATING.md#copy-the-database-data-over

**Pull request merged**